### PR TITLE
hooks: gi: add hooks for Rsvg

### PR DESCRIPTION
**Pull request by C0rn3j**
Fixes ~one of~ all the issues described in https://github.com/orgs/pyinstaller/discussions/8938

### CODE REVIEW

File: PyInstaller/hooks/hook-gi.repository.Rsvg.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2023, PyInstaller Development Team.
```
Comment (rokm) [inline PyInstaller/hooks/hook-gi.repository.Rsvg.py]: Copyright (c) 2024, PyInstaller Development Team.

**Comment by rokm** [comment]
Please add a [news fragment](https://pyinstaller.org/en/stable/development/changelog-entries.html) for the changelog (you can look at #8149 for examples).

File: PyInstaller/hooks/hook-gi.repository.Rsvg.py (hunk 1 above)
Comment (rokm) [reply PyInstaller/hooks/hook-gi.repository.Rsvg.py]: Ah, sorry - I meant that for both hook files.

**Comment by C0rn3j** [comment]
Should be good to go

**Review by rokm — APPROVED** [review]

**Comment by rokm** [comment]
Thank you!

**Pull request merged**